rootfs/etc/cont-init.d/14-config-rspamd.sh

```diff
@@ -94,10 +94,15 @@ password = "${REDIS_PASSWORD}";
 read_servers = "${REDIS_HOST}";
 EOL
 
+if [ "$RSPAMD_GREYLIST_ENABLE" = true ]; then
   echo "Setting Rspamd greylist.conf"
   cat >/etc/rspamd/local.d/greylist.conf <<EOL
 servers = "${REDIS_HOST}:${REDIS_PORT}";
 EOL
+else
+  echo "Rspamd greylisting disabled"
+  echo "enabled = false;" > /etc/rspamd/local.d/greylist.conf
```

Comment:
Pushed extra commit to fix the location based on official docs https://docs.rspamd.com/faq/#how-can-i-disable-greylisting